pom.xml

```diff
@@ -220,6 +220,7 @@
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast</artifactId>
             <version>${hazelcast.version}</version>
+            <scope>provided</scope>
```

Comment:
When we add that then no hazelcast will be pulled in into development environments right? It might not be a good experience...

Reply:
Discussed offline, in general `hazelcast-aws` release cycle is different than `hazelcast` and therefore it should not be automatically fetched. Otherwise there is an issue when both `hazelcast-aws` and `hazelcast-enterprise` are in the Maven dependencies.

Reply:
I see your point. lets add this change to the release notes and mention about potential impact on existing users.